modules/stepper/demo/stepper.html

```diff
@@ -0,0 +1,127 @@
+    </lx-component>
+
+    <lx-component lx-title="Optional steps" lx-path="/includes/modules/stepper/includes/optional.html" lx-js-path="/js/stepper/demo/demo-stepper_controller.js">
+        Optional steps within a linear flow should be marked as optional (Only if linear stepper).
+    </lx-component>
+
+    <lx-component lx-title="Error state" lx-path="/includes/modules/stepper/includes/error.html" lx-js-path="/js/stepper/demo/demo-stepper_controller.js"></lx-component>
+
+    <lx-component-attributes lx-title="Stepper attributes">
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Default</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                <tr>
+                    <td>lx-cancel*</td>
+                    <td><code>function</code></td>
+                    <td></td>
+                    <td>Cancel function called on cancel button click.</td>
+                </tr>
+                <tr>
+                    <td>lx-complete*</td>
+                    <td><code>function</code></td>
+                    <td></td>
+                    <td>Complete function called when stepper is complete.</td>
+                </tr>
+                <tr>
+                    <td>lx-is-linear</td>
+                    <td><code>boolean</code></td>
+                    <td><code>true</code></td>
+                    <td>Define the stepper type.</td>
+                </tr>
+                <tr>
+                    <td>lx-labels</td>
+                    <td><code>object</code></td>
+                    <td><code>{ 'back': 'Back', 'cancel': 'Cancel', 'continue': 'Continue', 'optional': 'Optional' }</code></td>
+                    <td>The buttons and the optionnal step labels.</td>
+                </tr>
+                <tr>
+                    <td>lx-layout</td>
+                    <td><code>string</code></td>
+                    <td><code>horizontal</code></td>
+                    <td>Define the stepper layout. Options: <code>horizontal</code>, <code>veritcal</code></td>
+                </tr>
+            </tbody>
+        </table>
+    </lx-component-attributes>
+
+    <lx-component-attributes lx-title="Step attributes">
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Default</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                <tr>
+                    <td>lx-feedback</td>
+                    <td><code>string</code></td>
+                    <td></td>
+                    <td>The feedback string displayed during the next step loading (Only if horizontal layout).</td>
```

Comment:
'The feedback string displayed while the next step is loading (Only if horizontal layout).'